test/GetIntrinsic.js

```diff
@@ -34,6 +34,36 @@ test('throws', function (t) {
 		'"just a dot" intrinsic throws a syntax error'
 	);
 
+	t['throws'](
+		function () { GetIntrinsic('%String'); },
+		SyntaxError,
+		'Leading % without trailing % throws a syntax error'
+	);
+
+	t['throws'](
+		function () { GetIntrinsic('String%'); },
+		SyntaxError,
+		'Trailing % without leading % throws a syntax error'
+	);
```

Comment:
I consider these bug fixes; it'd be great to put them in their own commit so i can pull that in as a patch.